src/pages/edit-profile/edit-profile.tsx

```diff
@@ -14,101 +16,107 @@ export const EditProfilePage: React.FunctionComponent = () => {
           height: 'calc(100vh - 65.3px)',
         }}
       >
-        <Box
-          sx={{
-            display: 'flex',
-            alignItems: 'center',
-            flexDirection: 'column',
-            width: 1,
-            height: '550px',
-            backgroundColor: '#e8eaf6',
-            borderRadius: '33px',
-          }}
-        >
-          <form>
-            <Box
-              sx={{
-                display: 'flex',
-                alignItems: 'center',
-                flexDirection: 'column',
-                width: 1,
-                mt: '10px',
-              }}
-            >
-              <Typography variant="h4" color="initial">
-                Edit Profile
-              </Typography>
-              <TextField
+        {isAuth ? (
+          <Box
+            sx={{
+              display: 'flex',
+              alignItems: 'center',
+              flexDirection: 'column',
+              width: 1,
```

Comment:
width: 1 ?

Reply:
<Box sx={{ width: 1 }}> // 100%   why not?